presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -159,6 +163,34 @@ public static double toUnixTimeFromTimestampWithTimeZone(@SqlType(StandardTypes.
         return unpackMillisUtc(timestampWithTimeZone) / 1000.0;
     }
 
+    @ScalarFunction("to_iso8601")
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice toISO8601(ConnectorSession session, @SqlType(StandardTypes.TIMESTAMP) long timestamp)
+    {
+        DateTimeFormatter formatter = ISODateTimeFormat.dateTime()
+                .withChronology(getChronology(session.getTimeZoneKey()));
+        return utf8Slice(formatter.print(new DateTime(timestamp)));
+    }
+
+    @ScalarFunction("to_iso8601")
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice toISO8601FromTimestampWithTimeZone(@SqlType(StandardTypes.TIMESTAMP_WITH_TIME_ZONE) long timestampWithTimeZone)
+    {
+        long millisUtc = unpackMillisUtc(timestampWithTimeZone);
+        DateTimeZone dateTimeZone = unpackDateTimeZone(timestampWithTimeZone);
+        return utf8Slice(ISODateTimeFormat.dateTime().print(new DateTime(millisUtc, dateTimeZone)));
```

Comment:
Same here.  Just search the code base for other uses of `DateTimeFormatter.print`.